R/epi.R

```diff
@@ -54,8 +58,8 @@ dalys <- function(
     mild_disability_weight = 0.006,
     moderate_disability_weight = 0.051,
     severe_disability_weight = 0.133,
-    clinical_episode_length = 0.01375,
-    severe_episode_length = 0.04795
+    clinical_episode_length = 0.01375 * 365,
```

Comment:
These should not be multiplied by 365. The output of the DALYs is in years and the units of these parameters are in years already. 
E.g. The length of a clinical episode is ~ 5 days = 0.01375 years, so the number of diability adjusted life _years_ is cases * weight * 0.1375 years.